js/modules/Debug/Debug.js

```diff
@@ -0,0 +1,1389 @@
+                                <thead>
+                                    <tr>
+                                        <th>${_x('debug', 'Number')}</th>
+                                        <th style="max-width: 200px; white-space: pre-wrap;">${_x('debug', 'URL')}</th>
+                                        <th>${_x('debug', 'Status')}</th>
+                                        <th>${_x('debug', 'Type')}</th>
+                                        <th>${_x('debug', 'Duration')}</th>
+                                    </tr>
+                                </thead>
+                                <tbody style="white-space: nowrap">
+                                </tbody>
+                            </table>
+                        </div>
+                    </div>
+                    <div class="resize-handle"></div>
+                    <div class="right-panel overflow-auto ms-2 flex-grow-1">
+                        <div id="debugpanel${rand}" class="p-0 mt-n1">
+                            <ul class="nav nav-tabs" data-bs-toggle="tabs">
+                                <li class="nav-item">
+                                    <button class="nav-link" data-bs-toggle="tab" data-glpi-debug-widget-id="request_summary">${_x('debug', 'Summary')}</button>
+                                </li>
+                                <li class="nav-item">
+                                    <button class="nav-link" data-bs-toggle="tab" data-glpi-debug-widget-id="sql">${_x('debug', 'SQL')}</button>
+                                </li>
+                                <li class="nav-item">
+                                    <button class="nav-link" data-bs-toggle="tab" data-glpi-debug-widget-id="globals">${_x('debug', 'Globals')}</button>
+                                </li>
+                                <li class="nav-item">
+                                    <button class="nav-link" data-bs-toggle="tab" data-glpi-debug-widget-id="profiler">${_x('debug', 'Profiler')}</button>
+                                </li>
+                            </ul>
+                
+                            <div class="card-body overflow-auto p-1">
+                                <div class="tab-content request-details-content-area">
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            `);
+            this.showRequestTimeline(content_area.find('.request-timeline').eq(0));
+            const truncated_pathname = window.location.pathname.substring(0, this.REQUEST_PATH_LENGTH);
+            content_area.find('#debug-requests-table tbody').append(`
+                <tr data-request-id="${this.initial_request.id}" class="cursor-pointer table-active">
+                    <td>0</td>
+                    <td style="max-width: 200px; white-space: pre-wrap;" title="${window.location.pathname}">${truncated_pathname}</td>
```

Comment:
Maybe a small enhancement can be added here, for people that would like to copy/past the full URL.
We could add a `...` badge that would permit to show the full URL when it is clicked.
![image](https://github.com/glpi-project/glpi/assets/33253653/385d3183-9de6-456a-8b3f-e53b5cfa6704)